cleint.h

```diff
@@ -0,0 +1,21 @@
+#include<iostream>
+#include"shop.h"
+#include"reus_code.h"   
+
+namespace cleint
+{
+    class Cleint
+    {
+    public:
+        void operation(abs_shop::Shop *choosen){
+            choosen->display_menu();
+            std::cout<<"enter product num = 0 to finish order\n";
+            UI::take_order(choosen);
```

Comment:
let  take_order() to abstracted method in shop
so we can call it like 
choosen->take_order()  because every shop differ than other and if developer work on specific shop 
no need to modify any thing in code  rather than  child shop class